site/src/css/custom.css

```diff
@@ -359,14 +363,15 @@ ul.table-of-contents li > ul li {
   max-width: none;
 }
 
+
```

Comment:
This extra blank line is what's causing the linting check to fail on CI.

(You can configure VSCode to use Prettier to auto-format your files whenever you save them. That can avoid these kinds of issues)